elog_zulip/__init__.py

```diff
@@ -264,13 +328,16 @@ def _publish(self, text, attributes, attachments, maxchar=10_000):
 
         # upload attachments
         attachments_text = ''
+        attachments_by_filename = {}
```

Comment:
is this used for anything?

Reply:
It is used to fix links documents that were included in the text but were not updated to the location of the uploaded file. See lines below (340, 359, 367..)